zanata-war/src/main/webapp-jboss/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -3,12 +3,15 @@
 <!-- https://access.redhat.com/knowledge/docs/en-US/JBoss_Enterprise_Application_Platform/6/html/Development_Guide/Included_Modules1.html -->
 <jboss-deployment-structure xmlns="urn:jboss:deployment-structure:1.2">
   <deployment>
-    <!-- These exclusions let us use RESTEasy 2 instead of the
+    <!-- These exclusions let us use the war file's RESTEasy instead of the
          container's version -->
     <exclude-subsystems>
       <subsystem name="jaxrs" />
     </exclude-subsystems>
     <exclusions>
+      <module name="javax.ws.rs.api" />
+      <module name="javaee.api" />
```

Comment:
Excluding this causes the class not found exception (missing "javax.inject-api"). We either declare a jboss module dependency "javax.inject.api" or we include the jar in war.